src/dndbeyond/content-scripts/character.js

```diff
@@ -369,6 +369,17 @@ function rollItem(force_display = false) {
                     brutal += 1;
             }
         }
+
+        //Artificer Battlemaster Arcane Jolt
+        // TODO: Implement for Steel Defender at later date
+        if (damages.length > 0 &&
+            character.hasClassFeature("Arcane Jolt") &&
+            character.getSetting("artificer-arcane-jolt", false) &&
+            item_type.indexOf(", Common") === -1) {
```

Comment:
😬 I'm unsure about the ", Common" thing (I think it already broke since now it's ", common" lowercase.
How about checking for "magic" in the description? I noticed most/all magic weapons will say "this magic <thing>" in its description.

Reply:
Yeah, I'll have a look again, I just remember coding this and going "how the crap are you supposed to figure out what to apply this to..."

Reply:
Honestly, I think checking for anything but it being not common isn't going to work. I found discrepancies between Named weapons and +X weapons, as well as just named weapons with no +X to hit and attack.

Testing for not-common means anything magical will get the application, as they're magical.